scripts/config/qconf.h

```diff
@@ -116,7 +116,7 @@ public slots:
 	enum optionMode optMode;
 	struct menu *rootEntry;
 	QPalette disabledColorGroup;
-	QPalette inactivedColorGroup;
+	QPalette unusedInactiveColorGroup;
```

Comment:
This is the only really drastic change. It's more of an anchor for me to make this comment:

This variable appears to be entirely unused.

`deactivated` is a word as is `inactive`. I'm not entirely sure what the goal of this object is, and I couldn't figure it out as I couldn't find anyone referencing it.